hub/config/src/main/java/uk/gov/ida/hub/config/application/CertificateService.java

```diff
@@ -50,9 +58,16 @@ public  Certificate encryptionCertificateFor(String entityId) {
 
     public List<Certificate> signatureVerificationCertificatesFor(String entityId) {
         T config = getConfig(entityId);
-        return config.getSignatureVerificationCertificates()
+
+        Map<Boolean, List<Certificate>> certsByValidity = config.getSignatureVerificationCertificates()
+                .stream()
+                .collect(partitioningBy(cd -> certificateValidityChecker.isValid(cd)));
+
+        certsByValidity.get(false)
+                .forEach(cd -> LOG.warn("Signature verification certificates were requested for entityId '{}' but at least one is invalid", entityId));
```

Comment:
I think we could word the message better now. We know how many of the certs are invalid at this stage. We could also make sure we don't log more than one message.
something like:
```
int badCertCount = certsByValidity.get(false).size();
        if (badCertCount > 0){
            LOG.warn("Signature verification certificates were requested for entityId '{}' {} invalid", entityId, badCertCount == 1 ? "one is" : "both are");
        }
```